test/stdlib/StringReallocation.swift

```diff
@@ -1,6 +1,8 @@
 // RUN: %target-run-simple-swift | %FileCheck %s
 // REQUIRES: executable_test
 
+import StdlibUnittest
```

Comment:
I moved some of the stdlib SPI out of the standard library and into StdlibUnittest. This test was using those SPI.